spec/fortnox/api/types/required_spec.rb

```diff
@@ -15,25 +15,27 @@
     end
 
     it 'raises an error' do
-      is_expected.to raise_error(Dry::Struct::Error, error_message)
+      expect(subject).to raise_error(Dry::Struct::Error, error_message)
```

Comment:
Same here about dropping the description and keeping the is_expected form.

Reply:
Also, if this can still be on the form where `subject` is expected to raise an error, why couldn't the last one ^?

Reply:
We should not have subject raising an error. See https://blog.rubystyle.guide/rspec/2019/07/17/rspec-implicit-block-syntax.html